hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/AdminServiceModule.java

```diff
@@ -14,19 +14,19 @@
  * limitations under the License.
  */
 
-package com.hedera.node.app.service.admin.impl.components;
+package com.hedera.node.app.service.admin.impl;
 
+import com.hedera.node.app.service.admin.impl.handlers.AdminHandlers;
 import com.hedera.node.app.service.admin.impl.handlers.FreezeHandler;
-import dagger.Component;
-import javax.inject.Singleton;
+import dagger.Module;
 
-@Singleton
-@Component
-public interface AdminComponent {
-    @Component.Factory
-    interface Factory {
-        AdminComponent create();
-    }
+/**
+ * Dagger module of the admin service
+ */
+@Module
+public interface AdminServiceModule {
 
     FreezeHandler freezeHandler();
```

Comment:
Should this method be removed now?

Reply:
That is something we should discuss once all services are refactored: Do we want to allow to inject individual handlers or only the `FooHandlers` component to access all handlers of a service?